app/models/post.rb

```diff
@@ -5,4 +5,16 @@ class Post < ApplicationRecord
 
   validates :title, presence: true
   validates :text, presence: true
+
```

Comment:
- [ ] Please set the update post counter method  to `private` and please use the `after_save` method  to update counter after a post has been saved to the database 👍🏿 

```
 after_save :update_post_counter

  private
```